apps/judicial-system/web/src/utils/hooks/useEventLog/index.ts

```diff
@@ -0,0 +1,35 @@
+import { useCallback } from 'react'
+import { useIntl } from 'react-intl'
+import { toast } from 'react-toastify'
+
+import { errors } from '@island.is/judicial-system-web/messages'
+import { CreateEventLogInput } from '@island.is/judicial-system-web/src/graphql/schema'
+
+import { useCreateEventLogMutation } from './createEventLog.generated'
+
+const useEventLog = () => {
+  const { formatMessage } = useIntl()
+  const [createEventLogMutation] = useCreateEventLogMutation()
+
+  const createEventLog = useCallback(
```

Comment:
Consider always returning a result (boolean?) so that the caller can verify the calls success. See comment above.